### PR TITLE
WIP: Installation documentation setup

### PR DESCRIPTION
**Pull request by ovv**
Issue #329

### CODE REVIEW

File: docs/setup.rst [hunk 1]
```diff
@@ -39,34 +39,39 @@ or just download the tarball and go that route:
...
-.. _bare metal: setup-installation-standard_
+You can go multiple routes with setting up and running Paperless:
+
+ * The `bare metal route`_
+ * The `vagrant route`_
+ * The `docker route`_
+
+
+The `vagrant route`_ is quick & easy, but means you're running a VM which comes
+with memory consumption, cpu overhead etc. The `docker route`_ offer the same
```
Comment (danielquinn) [inline docs/setup.rst]: Should be "offers"

File: docs/setup.rst [hunk 2]
```diff
@@ -39,34 +39,39 @@ or just download the tarball and go that route:
...
+simplicity a vagrant with less ressources consumption. Only the native usage
```
Comment (danielquinn) [inline docs/setup.rst]: * "a vagrant" should be "as Vagrant"
* "less ressources" should be "lower resource"
* I don't understand what you're trying to say with `Only the native usage under linux is documented you will need to adapt the documentation when using Docker Machine`.

File: docs/setup.rst [hunk 3]
```diff
@@ -39,34 +39,39 @@ or just download the tarball and go that route:
...
++++++++++++++++++++++
 
 1. Install the requirements as per the :ref:`requirements <requirements>` page.
 2. Within the extract of master.zip go to the ``src`` directory.
-3. Copy ``../paperless.conf.example`` to ``/etc/paperless.conf`` also the virtual
-   envrionment look there for it and open it in your favourite editor.
-   Because this file contains passwords it should only be readable by user root
-   and paperless !  Set the values for:
+3. Copy ``../paperless.conf.example`` to ``/etc/paperless.conf``. This file
+   contains passwords it should only be readable by user root and paperless !
```
Comment (danielquinn) [inline docs/setup.rst]: This should probably be "...contains passwords.  It should..."

File: docs/setup.rst [hunk 4]
```diff
@@ -92,13 +98,18 @@ Standard (Bare Metal)
 10. Visit the document list on your webserver, and it should be there, indexed
     and downloadable.
 
-.. _Paperless webserver: http://127.0.0.1:8000
+.. caution::
 
+    This installation is not secure. Once everything is working head up to
```
Comment (danielquinn) [inline docs/setup.rst]: "head up to" implies scrolling up.  Better to say "head over to".

File: docs/setup.rst [hunk 5]
```diff
@@ -292,49 +303,161 @@ Vagrant Method
...
+job well enough on a home network. However it is heavily discouraged to use
+it for more than that.
+
+If you want to do things right you should use a real webserver capable of handling
+more than one thread. You will also have to let the webserver serve the static
+files (CSS, JavaScript) from the directory configured in ``PAPERLESS_STATICDIR``.
+The default static files directory is ``../static``.
+
+For that you need to activate your virtual environment and collect the static
+file with the command:
```
Comment (danielquinn) [inline docs/setup.rst]: * "file" should be "files"

**Comment by ovv** [comment]
Awesome thanks. I'm not a native english speaker so that helps a lot.

**Comment by danielquinn** [comment]
I had some extra time tonight, so I decided to fix the conflict and merge the rest of your changes.  Thanks so much for your contribution <3!

**Comment by ovv** [comment]
I kept trying to come back to this PR but couldn't find the time to do so. I'm glad you found the time to improve & merge it. Keep up the good work :+1:

**Pull request merged**